apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/anonymous/AEPSampleAppNewArchEnabled/MainApplication.kt

```diff
@@ -0,0 +1,101 @@
+import expo.modules.ReactNativeHostWrapper
+
+import java.util.Arrays
+//MainApplication.java
+import com.adobe.marketing.mobile.AdobeCallback
+import com.adobe.marketing.mobile.Assurance
+import com.adobe.marketing.mobile.Edge
+import com.adobe.marketing.mobile.Extension
+import com.adobe.marketing.mobile.Lifecycle
+import com.adobe.marketing.mobile.LoggingMode
+import com.adobe.marketing.mobile.Messaging
+import com.adobe.marketing.mobile.MobileCore
+import com.adobe.marketing.mobile.Places
+import com.adobe.marketing.mobile.Signal
+import com.adobe.marketing.mobile.Target
+import com.adobe.marketing.mobile.UserProfile
+import com.adobe.marketing.mobile.edge.bridge.EdgeBridge
+import com.adobe.marketing.mobile.edge.consent.Consent
+import com.adobe.marketing.mobile.edge.identity.Identity
+import com.adobe.marketing.mobile.optimize.Optimize
+
+
+
+class MainApplication : Application(), ReactApplication {
+
+  override val reactNativeHost: ReactNativeHost = ReactNativeHostWrapper(
+        this,
+        object : DefaultReactNativeHost(this) {
+          override fun getPackages(): List<ReactPackage> {
+            // Packages that cannot be autolinked yet can be added manually here, for example:
+            // packages.add(new MyReactNativePackage());
+            return PackageList(this).packages
+          }
+
+          override fun getJSMainModuleName(): String = ".expo/.virtual-metro-entry"
+
+          override fun getUseDeveloperSupport(): Boolean = BuildConfig.DEBUG
+
+          override val isNewArchEnabled: Boolean = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED
+          override val isHermesEnabled: Boolean = BuildConfig.IS_HERMES_ENABLED
+      }
+  )
+
+  override val reactHost: ReactHost
+    get() = ReactNativeHostWrapper.createReactHost(applicationContext, reactNativeHost)
+
+  override fun onCreate() {
+    super.onCreate()
+
+      MobileCore.setApplication(this);
+    MobileCore.setLogLevel(LoggingMode.DEBUG)
+    MobileCore.configureWithAppID("94f571f308d5/12dfd4531236/launch-7560fa0e7fdd-development")
+    val extensions: List<Class<out Extension?>> = Arrays.asList(
+      Lifecycle.EXTENSION,
+      Signal.EXTENSION,
+      Edge.EXTENSION,
+      Identity.EXTENSION,
+      Consent.EXTENSION,
+      EdgeBridge.EXTENSION,
+      Messaging.EXTENSION,
+      UserProfile.EXTENSION,
+      Assurance.EXTENSION,
+      Places.EXTENSION,
+      Target.EXTENSION,
+      Optimize.EXTENSION,
+      com.adobe.marketing.mobile.Identity.EXTENSION
+    )
+    MobileCore.registerExtensions(extensions,
+      AdobeCallback { o: Any? ->
+        MobileCore.lifecycleStart(
+          null
+        )
+      })
+    SoLoader.init(this, false)
+    if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+      // If you opted-in for the New Architecture, we load the native entry point for this app.
+      load()
+    }
+    ApplicationLifecycleDispatcher.onApplicationCreate(this)
+  }
+
+  override fun onConfigurationChanged(newConfig: Configuration) {
```

Comment:
https://developer.adobe.com/client-sdks/home/base/mobile-core/lifecycle/#register-lifecycle-with-mobile-core-and-add-appropriate-startpause-calls
Override `onResume` and `onPause` functions and add the following:
```java
 @Override
    public void onResume() {
        MobileCore.setApplication(getApplication());
        MobileCore.lifecycleStart(null);
    }

    @Override
    public void onPause() {
        MobileCore.lifecyclePause();
    }

```

Reply:
done